2018-11-25 14:03:58,142 ERROR: Exception on /user/whereat [GET] [in /Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/evh111/Desktop/programz/petblog/app/main/routes.py", line 76, in user
    next_url=next_url, prev_url=prev_url)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/evh111/Desktop/programz/petblog/app/templates/user.html", line 19, in template
    <p><a href="{{ https://en.gravatar.com/ }}">{{ _('Change your profile picture') }}</a></p>
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 212, in parse_if
    node.body = self.parse_statements(('name:elif', 'name:else',
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/parser.py", line 876, in subparse
    self.stream.expect('variable_end')
  File "/Users/evh111/Desktop/programz/petblog/myenv/lib/python3.7/site-packages/jinja2/lexer.py", line 384, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got ':'
2018-11-25 14:04:13,611 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:05:27,047 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:05:59,176 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:08:13,358 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:09:11,980 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:11:32,799 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:12:34,204 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:13:10,718 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:13:33,306 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:14:43,255 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:15:08,323 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
2018-11-25 14:16:52,928 INFO: Microblog startup [in /Users/evh111/Desktop/programz/petblog/app/__init__.py:82]
